source/game/g_utils.cpp

```diff
@@ -1872,8 +1868,7 @@ static bool G_LoadFiredefFromFile( int weapon, firedef_t *firedef ) {
 		return false;
 	}
 
-	Q_snprintfz( filename, sizeof( filename ), "weapondefs/%s %s.def", GS_FindItemByTag( weapon )->shortname,
-				 ( firedef->fire_mode == FIRE_MODE_STRONG ) ? "strong" : "weak" );
+	Q_snprintfz( filename, sizeof( filename ), "weapondefs/%s .def", GS_FindItemByTag( weapon )->shortname );
```

Comment:
tbh delete this whole WEAPONDEFS_FROM_DISK thing

Reply:
Done